instrumentation/runtime-metrics/javaagent/src/main/java/io/opentelemetry/instrumentation/javaagent/runtimemetrics/RuntimeMetricsInstaller.java

```diff
@@ -30,6 +31,7 @@ public void afterAgent(Config config, AutoConfiguredOpenTelemetrySdk unused) {
         .isInstrumentationEnabled(Collections.singleton("runtime-metrics"), DEFAULT_ENABLED)) {
 
       Classes.registerObservers(GlobalOpenTelemetry.get());
+      Cpu.registerObservers(GlobalOpenTelemetry.get());
```

Comment:
This could certainly happen as a follow-up item, but do we want to consider a configurable way of disabling CPU metrics for some users if they don't want it on by default?

Reply:
That's what views are for!